src/main/java/org/terasology/web/webSocket/WsHandler.java

```diff
@@ -14,37 +14,67 @@
  * limitations under the License.
  */
 
-package org.terasology.web.servlet;
+package org.terasology.web.webSocket;
 
 import java.io.IOException;
 
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonSyntaxException;
 import org.eclipse.jetty.websocket.api.Session;
 import org.eclipse.jetty.websocket.api.WebSocketAdapter;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.terasology.web.io.ActionResult;
+import org.terasology.web.io.JsonSession;
 
 /**
  * Manages one websocket session
  */
-public class EventSocket extends WebSocketAdapter {
+public class WsHandler extends WebSocketAdapter {
 
-    private static final Logger logger = LoggerFactory.getLogger(EventSocket.class);
+    private static final Logger logger = LoggerFactory.getLogger(WsHandler.class);
+    private static final Gson GSON = new GsonBuilder().disableHtmlEscaping().create();
```

Comment:
Would it make sense to have one common GSON instance that is used in both cases?